src/import_edf.jl

```diff
@@ -382,10 +651,18 @@ function store_edf_as_onda(edf::EDF.File, onda_dir, recording_uuid::UUID=uuid4()
     mkpath(joinpath(onda_dir, "samples") * '/')
 
     signals = Onda.Signal[]
-    edf_samples, diagnostics = edf_to_onda_samples(edf; custom_extractors=custom_extractors)
-    for e in diagnostics.errors
-        @warn sprint(showerror, e)
+    edf_samples, plan = edf_to_onda_samples(edf; kwargs...)
+    
+    errors = _get(Tables.columns(plan), :errors)
```

Comment:
As it stands, this will always be `missing` because the column is actually called `error`. (I had to go back and recheck haha)
```suggestion
    errors = _get(Tables.columns(plan), :error)
```

Reply:
hoo boy, good catch there.